stats/pp_stage_goal.php

```diff
@@ -46,11 +46,10 @@
 $title = _("MTD Pages");
 $x_title = "PP = " . $goal_percent . "% of $round_before_PP";
 
-// If PP is higher than round_before_PP, use greens, else reds
 if ($pp_pages_total > $pp_page_goal) {
-    $barColors = ["darkgreen"];
+    $barColors = ["graph-normal-series"];
 } else {
-    $barColors = ["darkred"];
+    $barColors = ["graph-above-goal"];
 }
```

Comment:
I am totally confused -- the changes seem to be saying that the `pp_stage_goal` graph colors will change to grey and a different red (which would likely confuse those that are used to green and red -- I don't think there's a problem with changing the shade of red, but they're used to green meaning that PPers have posted more pages in a given month than F2 has saved). However, if I look at the graph on the AH in your sandbox, I see the correct colors. What am I missing?

Reply:
The news items are still pointing to a different sandbox so those colors are still old. As for the green I can revert to the exact shade if you want. I wasn't sure if we wanted more consistency or not.

Reply:
> The news items are still pointing to a different sandbox so those colors are still old. As for the green I can revert to the exact shade if you want. I wasn't sure if we wanted more consistency or not.

Well, duh! 🤦‍♀️ 

Okay, I've changed them to point to the right sandbox. I keep forgetting that's a thing. However, I'm still confused. 

From `layout.less`:

``` css
@graph-normal-series: #eeeeee;
@graph-above-goal: #ff484f;
```

* #eeeeee; is a very light grey and #ff484f; is a red. The round backlog graphs show this correctly,  but `pp_stage_goal` does not. The graph in your sandbox shows F2 in grey and PP in blue, whereas I expected them to be both grey or both red from how I was reading the code (see next bullet point). The blue came as a complete surprise.
* In addition, `pp_stage_goal` currently shows both bars green if the number of pages posted is greater than the number of pages saved as done in F2, and red if the number of pages saved in F2 is greater than the number of pages posted. The graph in your sandbox shows two different colors. (I also don't understand how that happened, because it looks like literally the only thing you changed here was the color. Maybe an earlier change that I didn't notice because the sandbox was pointing to the wrong version?)

Reply:
yeah, it was due to changes I made from barColors being actual colors to being class names. I have resolved it as well as moved bars to be above x axis because I think it looked quite strange. Line plots will still render above the x axis on zero.

Reply:
> yeah, it was due to changes I made from barColors being actual colors to being class names. I have resolved it as well as moved bars to be above x axis because I think it looked quite strange. Line plots will still render above the x axis on zero.

Looks good now.

However, can you take another look at the round graphs? It looks like something got messed up with the colors for "Goal" and "21-day SMA". "Actual" looks correct, but "Goal" should be orange and is shown as blue, and the SMA should be green and is shown as orange.